### DISCUSSION TITLE
Angående validering av attributtet 'healthcare_service' under 'care_relationship'

### DISCUSSION THREAD

**Discussion by oyvind-bech** [NorskHelsenett/Tillitsrammeverk#91] [General]
https://github.com/NorskHelsenett/Tillitsrammeverk/blob/main/specs/informasjons_og_datamodell.md#427-care_relation-behandlerrelasjon

Kodeverket «Tjenestetyper innen spesialisthelsetjenesten» er markert i dokumentet med Volven-OID 8655 (https://volven.no/produkt.asp?id=507419&catID=3&subID=8), som er beskrevet som «Helsehjelpsområde (OID=8655)».

Derimot har «Tjenestetyper innen spesialisthelsetjenesten» Volven-OID 8627. 

a) hva er korrekt?
b) det hadde vært fint hvis listen med kodeverk hadde vist OID-verdiene i klartekst

**Comment by richardhus** [comment]
Her må vi nok være litt pragmatiske i starten og også forholde oss til hva fagsystemene våre er i stand til å avgi på kort sikt uten at vi må sette opp mappinger. For DIPS-kunder er det faktisk kodeverket 8451 vi kan avgi koder fra i healthcare_service og som vi har brukt så langt under prøvedriften. Samtidig kan både 8655 og det tidligere 8627 være aktuelle fra andre enn DIPS-kunder, avhengig av hva deres systemer er i stand til å avgi på nåværende tidspunkt. Og kommunene har sine egne kodeverk igjen. Så det viktige med valideringen her er nok at dere kikker på oppgitt OID for koden som er oppgitt for å avklare at den er en gyldig kode. Vi må støtte flere ulike koderverk her, i alle fall i starten, og så må man eventuelt kjøre standardiseringsprosessene som fort tar flere år å komme i mål med hvis vi ser at det er et tydelig behov for å standardisere på færre kodeverk. Det viktige nå er at kodeverkene som brukes beskriver noenlunde det samme, selv om granuleringsnivået i kodeverket og hva opprinnelsen til kodeverket man bruker er kan være ulikt.

Når det gjelder b) er jeg enig i at vi gjerne må ha oversikten over kodeverkene som vil bli brukt med OID-verdiene for dem synlig i oversikten som ligger i informasjonsmodellen. Hvilke kodeverk som vil bli brukt vil bli avklart med den enkelte aktør/leverandør som eventuelt møter på at de ikke støtter noen av kodeverkene som så langt står i oversikten og dermed melder inn det kodeverket de kan forholde seg til. Det blir i alle tilfeller en begrenset mengde, selv om det nok er flere kodeverk enn ønskelig i en ideell verden.
